#!/usr/bin/env python
PACKAGE = "robosys_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("label_area", int_t, 0, "", 400, 200, 2500)
gen.add("circularity", int_t, 0, "", 70, 0, 100)

exit(gen.generate(PACKAGE, "robosys_ros", "Ball"))
